docs/pages/reference/core/createFrames.mdx

```diff
@@ -60,6 +76,8 @@ A secret key used to sign and verify the state in the frame message. If provided
   - If the secret is provided and signature does not match error is thrown.
   - If the secret is provided and signature matches, the state will be used as is.
 
+Also used to sign async image rendering payloads if `imagesRoute` is not `null`.
```

Comment:
Correct me if I'm wrong but I think the secret is used no matter the `imagesRoute` value, right?

Reply:
Ah right, setting the `imagesRoute` to `null` disables it completely.